src/ralph/sim_cards/models.py

```diff
@@ -0,0 +1,93 @@
+    pin1 = models.CharField(
+        max_length=8, null=True, blank=True, help_text=_('required numeric '
+                                                         'characters only '
+                                                         '(max length 16)'),
+        validators=[
+            RegexValidator(regex='^\d{1,8}$',
+                           message='required numeric '
+                                   'characters only (max length 8)'
+                           )
+        ]
+    )
+    puc1 = models.CharField(
+        max_length=16, help_text=_('required numeric '
+                                   'characters only '
+                                   '(max length 16)'),
+        validators=[
+            RegexValidator(regex='^\d{1,8}$',
+                           message='required numeric '
+                                   'characters only (max length 16)'
+                           )
+        ])
+    pin2 = models.CharField(
+        max_length=8, null=True, blank=True, help_text=_('asd'),
+        validators=[
+            RegexValidator(regex='^\d{1,8}$',
+                           message='required numeric '
+                                   'characters only (max length 8)'
+                           )
+        ]
+    )
+    puc2 = models.CharField(
+        max_length=16,
+        validators=[
+            RegexValidator(regex='^\d{1,8}$',
+                           message='required numeric '
+                                   'characters only (max length 16)'
+                           )
+        ])
+    carrier = models.ForeignKey(
+        CellularCarrier, on_delete=models.PROTECT,
+    )
+    card_number = models.CharField(
+        max_length=22, unique=True,
+        validators=[
+            RegexValidator(regex='^\d{1,22}$',
+                           message='required numeric '
+                                   'characters only (max length 22)'
+                           )
+        ]
+    )
+    phone_number = models.CharField(
+        max_length=16, unique=True, help_text=_('asd'),
+        validators=[
+            RegexValidator(regex='^[+]\d{1,16}$',
+                           message='required char "+" and numeric '
+                                   'characters only (max length 16), '
+                                   'ex. +48668234346'
+
+                           )
+        ]
+    )
+    warehouse = models.ForeignKey(Warehouse, on_delete=models.PROTECT)
+    owner = models.ForeignKey(
+        settings.AUTH_USER_MODEL, null=True, blank=True,
+        on_delete=models.SET_NULL,
+        related_name='owned_simcards',
+    )
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL, null=True, blank=True,
+        on_delete=models.SET_NULL,
+        related_name='used_simcards',
+    )
+
+    class Meta:
+        verbose_name = _('SIM Card')
+        verbose_name_plural = _('SIM Cards')
+
+    def __str__(self):
+        return _('SIM Card: {}').format(self.phone_number)
```

Comment:
``return self.phone_number`` will be enough

Reply:
After f2f discussion with @romcheg decided to leave this as is